static/app/views/settings/project/server-side-sampling/samplingPromo.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from '@emotion/styled';
```

Comment:
are we sure about prefixing all the files?

Reply:
Not all the files. I didn't want to pollute the global space by exporting `Promo` which is very generic. So I renamed the exported function component and also matched the filename.